samples/Actor/DemoActor/DemoActor.cs

```diff
@@ -17,18 +16,22 @@ namespace DaprDemoActor
     /// For Actors to use Reminders, it must derive from IRemindable.
     /// If you don't intend to use Reminder feature, you can skip implementing IRemindable and reminder specific methods which are shown in the code below.
     /// </summary>
-    public class DemoActor : Actor, IDemoActor, IRemindable
+    public class DemoActor : Actor, IDemoActor, IBankActor, IRemindable
     {
         private const string StateName = "my_data";
 
+        private readonly BankService bank;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="DemoActor"/> class.
         /// </summary>
         /// <param name="service">Actor Service hosting the actor.</param>
-        /// <param name="actorId">Actor Id.</param>
-        public DemoActor(ActorService service, ActorId actorId)
-            : base(service, actorId)
+        public DemoActor(ActorHost service, BankService bank)
```

Comment:
add param name comment for bank

Reply:
rename service to actorHost

Reply:
We should stop adding doc-comments on samples.

They are useful when someone is going to inspect the code in the IDE (libraries) or when the generated documentation is going to be uploaded somewhere. For sample code it's just busywork. Most doc-comments are boilerplate and say nothing of substance. If we have something to say about a sample we should do with a normal comment where it's more readable.

Will rename the parameter.

Reply:
where is this constructor used?